### PR TITLE
cli: add admin UI URL to demo message

### PR DESCRIPTION
**Pull request by madelynnblue**
Release note (cli change): show admin UI URL in demo message.

### CODE REVIEW

**Comment by mjibson** [comment]
bors r=jordanlewis

**Comment by mjibson** [comment]
(I thought bors would wait until an approved review from the listed user. My mistake.)

**Review by jordanlewis — APPROVED** [review]
Good idea!

**Pull request merged**
